src/parameterizations/vertical/MOM_tidal_mixing.F90

```diff
@@ -799,7 +802,11 @@ subroutine calculate_CVMix_tidal(h, j, G, GV, US, CS, N2_int, Kd_lay, Kv)
       do k=1,G%ke
         Kd_lay(i,j,k) = Kd_lay(i,j,k) + 0.5 * US%m2_s_to_Z2_T * (Kd_tidal(k) + Kd_tidal(k+1))
       enddo
-
+      if (present(Kd_int)) then
+        do k=1,G%ke+1
```

Comment:
Elsewhere in the MOM6 code, we try to use an uppercase K index for variables that are discretized on interfaces, and a lowercase k for variables that are discretized on layers.  Because Fortran is case-insensitive, this is a soft convention for human readability only, but it might be beneficial here.